### PR TITLE
Version checker

### PR DESCRIPTION
**Pull request by jiivan**

### CODE REVIEW

File: core/version/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+pub mod notifier {
+    use std::time::Duration;
+    use tokio::time;
+
+    use ya_persistence::executor::DbExecutor;
+
+    const UPDATE_CURL: &'static str = "curl xxx|bash";
+    const SILENCE_CMD: &'static str = "yagna update skip";
+    const VERSION: &'static str = env!("CARGO_PKG_VERSION");
```
Comment (tworec) [inline core/version/src/lib.rs:14]: see `ya_compile_time_utils::semver()`

File: core/version/src/db.rs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        }
+
+        /*
+        pub async fn clean(&self) {
+            // TODO
+        }
+        */
+
+        pub async fn pending_release(&self) -> Result<Option<DBRelease>> {
+            do_with_transaction(self.pool, move |conn| {
```
Comment (tworec) [inline core/version/src/db.rs:53]:
```suggestion
            readonly_transaction(self.pool, move |conn| {
```

File: core/version/src/lib.rs (hunk 1 above)
Comment (jiivan) [reply core/version/src/lib.rs:14]: resolved in #959

**Review by tworec — DISMISSED** [review]
lgtm

**Review by tworec — APPROVED** [review]
lgtm

**Pull request merged**